src/webui/www/private/views/preferences.html

```diff
@@ -2128,10 +2138,26 @@
             $("locale_select").value = selected;
         };
 
+        const updateColoSchemeSelect = () => {
+            const combobox = $("color_scheme_select");
+            const colorScheme = LocalPreferences.get("color_scheme");
+
+            if (!colorScheme || ((colorScheme !== "light") && (colorScheme !== "dark")))
+                combobox.options[0].selected = true;
+            else if (colorScheme === "light")
+                combobox.options[1].selected = true;
+            else
+                combobox.options[2].selected = true;
```

Comment:
```suggestion
            if (!colorScheme)
                combobox.options[0].selected = true;
            else if (colorScheme === "light")
                combobox.options[1].selected = true;
            else if (colorScheme === "dark")
                combobox.options[2].selected = true;
```

Reply:
The proposed change doesn't do the same. I want to treat garbage values as `auto`.

Reply:
>The proposed change doesn't do the same. I want to treat garbage values as auto.

I see. Or perhaps:
```suggestion
            if (colorScheme === "light")
                combobox.options[1].selected = true;
            else if (colorScheme === "dark")
                combobox.options[2].selected = true;
            else
                combobox.options[0].selected = true;
```